Initial Github workflow

gdey opened the pull request.

fantabulous-tech commented:
Let's use `https://github.com/fantabulous-tech/solo-adventurers-toolbox/tree/master/Inky/resources/app.asar.unpacked/main-process/ink/inklcate` so we don't have duplicates.

gdey committed:
Initial Github workflow

This workflow will compile the Ink Scripts/solo.ink into
the solo.ink.json and docs/solo.js files.

gdey commented:
@fantabulous-tech updated with requested changes
